src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -199,6 +201,6 @@ def load_command_table(self, _):
     with self.command_group('aks trustedaccess rolebinding', trustedaccess_role_binding_sdk, client_factory=cf_trustedaccess_role_binding) as g:
         g.custom_command('list', 'aks_trustedaccess_role_binding_list')
         g.custom_show_command('show', 'aks_trustedaccess_role_binding_get')
-        g.custom_command('create', 'aks_trustedaccess_role_binding_create')
-        g.custom_command('update', 'aks_trustedaccess_role_binding_update')
+        g.custom_command('create', 'aks_trustedaccess_role_binding_create_or_update')
+        g.custom_command('update', 'aks_trustedaccess_role_binding_create_or_update')
```

Comment:
Why are these being changed?

Reply:
Not sure how those got modified but reverted them back.